localstack/services/s3/s3_listener.py

```diff
@@ -1307,5 +1320,91 @@ def return_response(self, method, path, data, headers, response, request_handler
                 response.headers['Content-Encoding'] = 'gzip'
 
 
+def authenticate_presign_url(method, path, data=None, headers={}):
+    # ignored_headers conatins headers which don't get involved in signature calculations process
+    # these headers are being sent by the localstack by default.
+    ignored_headers = [
```

Comment:
nit: Let's make this a top-level constant/variable in this file as well:
```
IGNORED_PRESIGN_HEADERS = [...]
```

Reply:
Ohk